### ISSUE TITLE
Merged oneway transportation - reverse oneway

### ISSUE THREAD

**Issue by frodrigo** [openmaptiles/planetiler-openmaptiles#59]
Despite there is already a fix (#10 and #40) I still see reversed merged oneway street.

This two are merged, not same name, but same attributes.
https://www.openstreetmap.org/way/26213715 https://www.openstreetmap.org/way/16873649

And one is reversed.

**Comment by frodrigo** [comment]
I do some debug.

https://github.com/openmaptiles/planetiler-openmaptiles/blob/main/src/main/java/org/openmaptiles/layers/Transportation.java#L530

```java
      if (oneway instanceof Integer i && ONEWAY_VALUES.contains(i)) {
        item.attrs().put(LIMIT_MERGE_TAG, onewayId++);
      }
```

In my case, the if condition is never true. `oneway` is a `Long`. Not sure about the right fix to do.
Using `if (oneway != null)`, do the job.

**Comment by msbarry** [comment]
Thanks for digging into that @frodrigo - I'm fixing the fix in #70
